### PR TITLE
Release consensus 0.2.0.0

### PR DESCRIPTION
**Pull request by Jasagredo**
# Description

Release version 0.2.0.0 of the `ouroboros-consensus` bundle. See the [changelog](https://github.com/input-output-hk/ouroboros-network/tree/release-consensus-0.2.0.0/ouroboros-consensus/CHANGELOG.md#0200--2023-02-09).

### CODE REVIEW

File: ouroboros-consensus/CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,32 @@ process](./docs/ReleaseProcess.md).
...
+### Breaking
+
+- Remove redundant proxy argument for `ledgerDbTip`.
+
+- Removed the `idx` type variable on the `Mempool` and `MempoolSnapshot`
+  datatypes in favour of using `TicketNo` always.
+
+- `Ouroboros.Consensus.Node` and `Ouroboros.Consensus.Network` hierarchies of
+  modules where moved from `ouroboros-consensus` to
+  `ouroboros-consensus-diffusion` package.
```
Comment (jorisdral) [inline ouroboros-consensus/CHANGELOG.md:43]: I see no scriv entry for this, was this added manually?

**Review by jorisdral — APPROVED** [review]
LGTM

File: ouroboros-consensus/CHANGELOG.md (hunk 1 above)
Comment (jorisdral) [reply ouroboros-consensus/CHANGELOG.md:43]: My bad, must have missed it

**Review by dnadales — APPROVED** [review]
Thanks!

**Pull request merged**